pkg/fleet/daemon/daemon.go

```diff
@@ -264,7 +264,11 @@ func (d *daemonImpl) startInstallerExperiment(ctx context.Context, url string) (
 	defer d.refreshState(ctx)
 
 	log.Infof("Daemon: Starting installer experiment for package from %s", url)
-	err = bootstrap.InstallExperiment(ctx, d.env, url)
+	if runtime.GOOS == "windows" {
+		err = d.installer.InstallExperiment(ctx, url)
+	} else {
+		err = bootstrap.InstallExperiment(ctx, d.env, url)
```

Comment:
❓ question

What does the `bootstrap.InstallExperiment` do differently from `installer.InstallExperiment` to warrant using it instead on Linux?